packages/flutter/lib/src/material/chip.dart

```diff
@@ -1377,7 +1378,8 @@ class _RawChipState extends State<RawChip> with TickerProviderStateMixin<RawChip
         onTapDown: canTap ? _handleTapDown : null,
         onTapCancel: canTap ? _handleTapCancel : null,
         child: new AnimatedBuilder(
-          animation: new Listenable.merge(<Listenable>[selectController, enableController]),
+          animation: new Listenable.merge(
+              <Listenable>[selectController, enableController]),
```

Comment:
unexpected new wrapping?
If you want to wrap this, put a comma after the `]` and the `)` on the next line

Reply:
Fixed